packages/core/src/utils.ts

```diff
@@ -98,6 +98,15 @@ export function convertBigIntToByteArray(bigInt: bigint) {
   return result.reverse()
 }
 
+export function convertCircuitBigIntToString(input: bigint) {
+  let result: string;
+  while (input > 0) {
+    result += String.fromCharCode(Number(input % BigInt(256)))
+    input = input / BigInt(256)
+  }
+  return result
```

Comment:
I think we need to `reverse` the `result` here.

Reply:
Sorry, `reverse` is not required as we are diving by 256 (which gives last char) and appending to the string.

Would be nice to add a comment like `The proof bigInt is in big endian format and each iteration appends the last char - which is the first char in little-endian format`